tests/slsa_analyzer/build_tool/__snapshots__/test_poetry.ambr

```diff
@@ -0,0 +1,16 @@
+# serializer version: 1
+# name: test_get_build_dirs[mock_repo0]
+  list([
+    PosixPath('.'),
+  ])
+# ---
+# name: test_get_build_dirs[mock_repo1]
+  list([
+    PosixPath('.'),
```

Comment:
This seems to be a false positive. The problem is that `get_build_dirs` function is not checking if the detected directory has a valid build. This PR should fix this issue: https://github.com/oracle-samples/macaron/pull/135

Reply:
Thank you for fixing that.